### ISSUE TITLE
Getting a response of 500 when attempting to create asset with asset_class specified.

### ISSUE THREAD

**Issue by rithviknishad** [coronasafe/care#792]
1. Create new asset with `asset_class` attribute included with a valid value (eg: "ONVIF" / "HL7MONITOR")

**Expected behavior**
201 Response

**Screenshots**

![image](https://user-images.githubusercontent.com/25143503/170700869-5dd427dd-d532-4904-9171-eacffc3f3bf9.png)

However, endpoint works fine when `asset_class` is not included.

![image](https://user-images.githubusercontent.com/25143503/170701025-da06d0a1-1d8d-453a-bff4-30aaa5a20e62.png)

**Comment by rithviknishad** [comment]
@vigneshhari @iamsdas still getting 500 as response :/
Am I doing something wrong or...?

![image](https://user-images.githubusercontent.com/25143503/171834317-b5c65b96-dec5-469f-997f-4c221bb5f892.png)

**Comment by iamsdas** [comment]
@rithviknishad have you pulled the latest changes from the master branch? It seems to be working fine for me locally (using the Swagger UI). Also, check if you are providing the necessary input fields.

Here's the request body I used for testing:
```
{
  "asset_type": "INTERNAL",
  "location_object": {
    "facility": {
      "name": "facility #1"
    },
    "name": "al 1",
    "description": "string",
    "location_type": 10
  },
  "location": "18b21291-1c87-4bfc-8c02-f60c5ccc285f",
  "name": "asset name",
  "description": "string",
  "asset_class": "ONVIF",
  "is_working": true,
  "not_working_reason": "string",
  "serial_number": "string",
  "warranty_details": "string",
  "meta": {},
  "vendor_name": "string",
  "support_name": "string",
  "support_phone": "+919999999999",
  "support_email": "user@example.com",
  "qr_code_id": null
}
```
![Screenshot_20220603_155057](https://user-images.githubusercontent.com/26461855/171836380-7d3229b9-5fd5-46c4-8e83-6546336933e5.png)

**Comment by rithviknishad** [comment]
@iamsdas I'm using the staging API via proxy mentioned in `care_fe/package.json`

I'm yet to setup a local care BE on my machine.

![image](https://user-images.githubusercontent.com/25143503/171849660-749941b9-314e-494f-b7df-696037c63ac8.png)

- What I'm performing is:
```sh
curl 'https://careapi.coronasafe.in/api/v1/asset/' \
        -X POST \
        -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:100.0) Gecko/20100101 Firefox/100.0' \
        -H 'Accept: application/json, text/plain, */*' \
        -H 'Accept-Language: en-US,en;q=0.5' \
        -H 'Accept-Encoding: gzip, deflate, br' \
        -H 'Content-Type: application/json' \
        -H 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0b2tlbl90eXBlIjoiYWNjZXNzIiwiZXhwIjoxNjU0MjU4MjY5LCJqdGkiOiIwZjgyMTQ5Mzc4MjY0ZWUyYTJmNzVlNDFmYzllMmRjMSIsInVzZXJfaWQiOjIwMn0.gp1grpzzKDH3ZCPoRSI_XTLX-IoZnWDATUcZQpolwuU' \
        -H 'Origin: http://localhost:4000' \
        -H 'Connection: keep-alive' \
        -H 'Referer: http://localhost:4000/facility/4fa7d5bf-1862-4ebb-b2d1-8284d12185ac/assets/new' \
        -H 'Sec-Fetch-Dest: empty' \
        -H 'Sec-Fetch-Mode: cors' \
        -H 'Sec-Fetch-Site: same-origin' \
        --data-raw ' 
{                              
  "name": "Hello Asset :wave:",
  "asset_type": "INTERNAL",
  "asset_class": "ONVIF",
  "description": "",   
  "is_working": "true",    
  "not_working_reason": "",
  "serial_number": "",   
  "warranty_details": "",                            
  "location": "afa28ab1-53f5-4554-8cb0-77c1e5d58c37",
  "vendor_name": "", 
  "support_name": "", 
  "support_email": "",             
  "support_phone": "+917777777777",
  "qr_code_id": "",
  "cancelToken": {
    "promise": {}
  }
}'

```

- And I'm getting the following for the above.
![image](https://user-images.githubusercontent.com/25143503/171851175-e385b0d2-8d30-419c-889e-e7e3b61b5e1f.png)

- However, if I just remove `asset_class` attribute in the same above cURL command:
```sh
curl 'https://careapi.coronasafe.in/api/v1/asset/' \
        -X POST \
        -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:100.0) Gecko/20100101 Firefox/100.0' \
        -H 'Accept: application/json, text/plain, */*' \
        -H 'Accept-Language: en-US,en;q=0.5' \
        -H 'Accept-Encoding: gzip, deflate, br' \
        -H 'Content-Type: application/json' \
        -H 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0b2tlbl90eXBlIjoiYWNjZXNzIiwiZXhwIjoxNjU0MjU4MjY5LCJqdGkiOiIwZjgyMTQ5Mzc4MjY0ZWUyYTJmNzVlNDFmYzllMmRjMSIsInVzZXJfaWQiOjIwMn0.gp1grpzzKDH3ZCPoRSI_XTLX-IoZnWDATUcZQpolwuU' \
        -H 'Origin: http://localhost:4000' \
        -H 'Connection: keep-alive' \
        -H 'Referer: http://localhost:4000/facility/4fa7d5bf-1862-4ebb-b2d1-8284d12185ac/assets/new' \
        -H 'Sec-Fetch-Dest: empty' \
        -H 'Sec-Fetch-Mode: cors' \
        -H 'Sec-Fetch-Site: same-origin' \
        --data-raw '
{
  "name": "Hello Asset :wave:",
  "asset_type": "INTERNAL",
  "description": "",
  "is_working": "true",
  "not_working_reason": "",
  "serial_number": "",
  "warranty_details": "",
  "location": "afa28ab1-53f5-4554-8cb0-77c1e5d58c37",
  "vendor_name": "",
  "support_name": "",
  "support_email": "",
  "support_phone": "+917777777777",
  "qr_code_id": "",
  "cancelToken": {
    "promise": {}
  }
}'
```

- I'm getting a good response for that:
![image](https://user-images.githubusercontent.com/25143503/171851568-8201dd1e-38a0-4a1e-bd3a-367eb724f981.png)

**Comment by sainAk** [comment]
The asset_class expects an integer
Exception: https://sentry.io/organizations/coronasafe-network/issues/3306110898/

**Comment by rithviknishad** [comment]
Weren't the migrations run automatically? It's a char field in the last change.

**Comment by sainAk** [comment]
It's waiting for deployment https://github.com/coronasafe/care/deployments/activity_log?environment=Staging-egov

@mathew-alex @tomahawk-pilot can you update the staging to deploy automatically?

**Comment by tomahawk-pilot** [comment]
We can but let it be there. Since we have multiple instances being deployed to

**Comment by sainAk** [comment]
> We can but let it be there. Since we have multiple instances being deployed to

@tomahawk-pilot I'm asking auto deployments only for Staging-egov, If the review requirement could be removed for `Staging-egov` environment the FE development and testing won't be blocked by pending deployments.

**Comment by tomahawk-pilot** [comment]
> > We can but let it be there. Since we have multiple instances being deployed to
> 
> @tomahawk-pilot I'm asking auto deployments only for Staging-egov, If the review requirement could be removed for `Staging-egov` environment the FE development and testing won't be blocked by pending deployments.

Feel free to remove the review requirements for staging egov environment

**Comment by rithviknishad** [comment]
@iamsdas working now! Great :)
@sainAk will you be moving this to a new issue as the original issue is now solved.

**Comment by sainAk** [comment]
> > > We can but let it be there. Since we have multiple instances being deployed to
> > 
> > 
> > @tomahawk-pilot I'm asking auto deployments only for Staging-egov, If the review requirement could be removed for `Staging-egov` environment the FE development and testing won't be blocked by pending deployments.
> 
> Feel free to remove the review requirements for staging egov environment

I don't have the privileges for it.

**Comment by mathew-alex** [comment]
@sainAk - I have removed the review requirements for staging egov environment. 👍
